### PR TITLE
added release notes for 4.3.2

### PR DESCRIPTION
**Pull request by gsps1**

### CODE REVIEW

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -30,6 +30,44 @@ Cask Data Application Platform Release Notes
...
+
+
+Bug Fixes
+---------
+
+- :cask-issue:`CCDAP-6032` - Minimize master's local storage usage by deleting the temporary directories created on the cdap-master for programs as soon as programs are launched on the cluster.
+
+- :cask-issue:`CDAP-12682` - Fixed an issue where UI was looking for the wrong property for SSL port.
+
+- :cask-issue:`CDAP-12693` - Fixed a bug that causes PySpark fail to run with Spark 2 in local sandbox.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: .. to fail to run ...

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 2]
```diff
@@ -30,6 +30,44 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-12701` - Packaging the SLF4J Log with the plugins rather than depending on the system settings.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: what's the context for this? Can't really tell what the issue was here.

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 3]
```diff
@@ -30,6 +30,44 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-12731` - Fixed an issue with that caused HBase Sink to fail when used alongside other sinks, using the Spark execution engine.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: seems incomplete... perhaps remove **with**

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 4]
```diff
@@ -30,6 +30,44 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+
+`Release 4.3.2 <http://docs.cask.co/cdap/4.3.2/index.html>`__
+=============================================================
+New Features
+------------
+
+- :cask-issue:`CDAP-12771` - Added GCS connection to Data Prep.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst:39]: shouldn't there be an issue about S3 too?

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 5]
```diff
@@ -30,6 +30,44 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-12693` - Fixed a bug that causes PySpark fail to run with Spark 2 in local sandbox.
+
+- :cask-issue:`CDAP-12701` - Packaging the SLF4J Log with the plugins rather than depending on the system settings.
+
+- :cask-issue:`CDAP-12731` - Fixed an issue with that caused HBase Sink to fail when used alongside other sinks, using the Spark execution engine.
+
+- :cask-issue:`CDAP-12743` - Fixed a bug where the Scala Spark compiler has missing classes from Classloader, causing compilation failure.
+
+- :cask-issue:`CDAP-12752` - Fixed a bug that failed to run Spark program when Spark authentication is turned on.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: .. where Spark Programs failed to run...?

**Review by bdmogal — CHANGES_REQUESTED** [review]

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 4 above)
Comment (gsps1) [reply cdap-docs/reference-manual/source/release-notes.rst:39]: yes, added it

**Comment by gsps1** [comment]
@bdmogal addressed comments, please take a look again.

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 6]
```diff
@@ -54,15 +55,15 @@ Bug Fixes
 
 - :cask-issue:`CDAP-12682` - Fixed an issue where UI was looking for the wrong property for SSL port.
 
-- :cask-issue:`CDAP-12693` - Fixed a bug that causes PySpark fail to run with Spark 2 in local sandbox.
+- :cask-issue:`CDAP-12693` - Fixed a bug that causes PySpark to fail to run with Spark 2 in local sandbox.
 
-- :cask-issue:`CDAP-12701` - Packaging the SLF4J Log with the plugins rather than depending on the system settings.
+- :cask-issue:`CDAP-12701` - Fixed a bug that causes deployment of pipelines with condition plugins to fail on ambari clusters.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: ``Apache Ambari``

**Review by bdmogal — APPROVED** [review]
minor comment. Rest LGTM

**Pull request merged**